docs/Makefile

```diff
@@ -27,6 +27,7 @@ build:
 	python gen_doc_files.py
 	make html
 	mv _build/html unstable
+	cp -r src/img unstable/
```

Comment:
@wuhu This was missing so the images would be properly linked. Out of curiosity, would it be "more" correct to store the images in the `static` folder and link to them in the `.md` files using `_static/my_image.png`?